working-groups/learning/minutes/2019.06.11-meeting.md

```diff
@@ -0,0 +1,44 @@
+# Meeting 06/11 notes
```

Comment:
I think I'd use 2019/06/11, also last meeting date format is also in US format. Maybe better to switch to this way that's better to avoid confusion.

Reply:
Standard date format is 2019-06-11 (with hyphens); US format should ofc not be used.

Reply:
I updated all three documents to be consistently 2019-MM-DD